rfcs/0000-additional-rule-field.md

```diff
@@ -1,35 +1,27 @@
-# 0000: Name of RFC
+# 0000: Additional Rule Field
```

Comment:
I think it should be inside `rfcs/text` and have next id in the beginning, see examples in that folder